tutorials/dataframe/df106_HiggsToFourLeptons.C

```diff
@@ -153,21 +152,20 @@ void df106_HiggsToFourLeptons()
    // kinematics such as pT or pseudorapidity.
    // Muons uncertainties are negligible, as stated in https://atlas.web.cern.ch/Atlas/GROUPS/PHYSICS/PAPERS/MUON
    // Electrons uncertainties are evaluated based on the plots available in https://doi.org/10.48550/arXiv.1908.0
-   // The uncertainties are linearly interpolated to cover a range of pT values covered by the analysis.
-   const std::vector<double> x{5.50e3, 5.52e3, 12.54e3, 17.43e3, 22.40e3, 27.48e3, 30e3, 10000e3};
-   const std::vector<double> y{0.06628, 0.06395, 0.06396, 0.03372, 0.02441, 0.01403, 0, 0};
-   unsigned int N = x.size();
-   ROOT::Math::Interpolator inter(N, ROOT::Math::Interpolation::kLINEAR);
-   inter.SetData(x, y);
+   // The uncertainties are linearly interpolated, using the `TGraph::Eval()` method, to cover a range of pT values
+   // covered by the analysis.
+   double x[] = {5.50e3, 5.52e3, 12.54e3, 17.43e3, 22.40e3, 27.48e3, 30e3, 10000e3};
+   double y[] = {0.06628, 0.06395, 0.06396, 0.03372, 0.02441, 0.01403, 0, 0};
+   TGraph graph(std::size(x), x, y);
```

Comment:
```suggestion
   const std::vector<double> x = {5.50e3, 5.52e3, 12.54e3, 17.43e3, 22.40e3, 27.48e3, 30e3, 10000e3};
   const std::vector<double> y = {0.06628, 0.06395, 0.06396, 0.03372, 0.02441, 0.01403, 0, 0};
   TGraph graph(x.size(), x.data(), y.data());
```
I would not encourage the use of C-style array to users. Why not stick with the `std::vector<>`?